File: app/src/main/java/me/shadura/escposprint/EscPosPrintApp.java
package me.shadura.escposprint;

import android.content.Context;
import androidx.multidex.MultiDexApplication;

import com.hypertrack.hyperlog.HyperLog;

public class EscPosPrintApp extends MultiDexApplication {
    public static final String LOG_TAG = "ESCPOS";

    private static EscPosPrintApp instance;

    public static EscPosPrintApp getInstance() {
        return instance;
    }

    public static Context getContext() {
        return instance.getApplicationContext();
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;

        HyperLog.initialize(this);
    }
}

File: app/src/main/java/me/shadura/escposprint/EscPosPrintApp.kt
package me.shadura.escposprint

import android.content.Context
import androidx.multidex.MultiDexApplication

import com.hypertrack.hyperlog.HyperLog

class EscPosPrintApp : MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()
        instance = this

        HyperLog.initialize(this)
    }

    companion object {
        val LOG_TAG = "ESCPOS"

        lateinit var instance: EscPosPrintApp

        val context: Context
            get() = instance.applicationContext
    }
}
